docs/sources/enterprise/license-restrictions.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "License restrictions"
+description = "Grafana Enterprise license restrictions"
+keywords = ["grafana", "licensing", "enterprise"]
+weight = 8
++++
+
+# License restrictions
+
+Enterprise licenses are limited by the number of active users, a license expiration date, and the URL of the Grafana instance.
+
+**User limits**
+
+Grafana licenses allow for a certain number of active users per instance. An active user is any user that has signed in to Grafana within the past 30 days.
+
+In the context of licensing, each user is classified as either a viewer or an editor:
+
+- An editor is a user who has permission to edit and save a dashboard. Examples of editors are as follows:
+    - Grafana server administrators.
+    - Users who are assigned an organizational role of `Editor` or `Admin`.
+    - Users that have been granted `Admin` or `Edit` permissions at the dashboard or folder level. Refer to [Dashboard and folder permissions](https://grafana.com/docs/grafana/latest/permissions/dashboard_folder_permissions/).     
+- A viewer is a user with the `Viewer` role, which does not permit the user to save a dashboard.
+
+Restrictions are applied separately for viewers and editors.
+
+When the number of maximum active viewers or editors is reached, Grafana displays a warning banner.
+
+**Expiration date**
+
+The license expiration date is the date when a license is no longer active. As the license expiration date approaches, Grafana Enterprise displays a banner.
+
+**License URL**
+
+License URL is the root URL of your Grafana instance. The license will not work on an instance of Grafana with a different root URL.
+
+## Dashboard and folder permissions report
+
+This CSV report helps to identify users, teams, and roles that have been granted `Admin` or `Edit` permissions at the dashboard or folder level.
+
+To download the report (in CSV format) go to the **Server Admin** > **Licensing**. From the bottom of the page, select **Download report**.
+
+## Updating license restrictions
+
+To increase the number of licensed users within Grafana, extend a license, or change your licensed URL, contact [Grafana support](https://grafana.com/profile/org#support) or your Grafana Labs account team. They will update your license, which you can activate from within Grafana. Refer to [Activate a license process](https://grafana.com/docs/grafana/latest/enterprise/activate-license/).
```

Comment:
```suggestion
To increase the number of licensed users within Grafana, extend a license, or change your licensed URL, contact [Grafana support](https://grafana.com/profile/org#support) or your Grafana Labs account team. They will update your license, which you can activate from within Grafana. 

For instructions on how to activate your license once it is updated, refer to [Activate an Enterprise license](https://grafana.com/docs/grafana/latest/enterprise/activate-license/).
```

Reply:
Two things:
- The second link needs to be a relref to activate-license.md.
- The link text should exactly match the topic or heading that you are linking to whenever possible. Review the Grafana style guidelines on [Headings](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#headings) for more guidance.

Reply:
On line 42, change "once" to "after" to reserve the word _once_ for things that happen one time. Here, it really means _after_.